build_tools/cmake/iree_check_test.cmake

```diff
@@ -243,8 +243,8 @@ function(iree_check_test_suite)
   )
 
   if(NOT DEFINED _RULE_TARGET_BACKENDS AND NOT DEFINED _RULE_DRIVERS)
-    set(_RULE_TARGET_BACKENDS "vmla" "vulkan-spirv" "llvm-ir")
-    set(_RULE_DRIVERS "vmla" "vulkan" "llvm")
+    set(_RULE_TARGET_BACKENDS "vmla" "vulkan-spirv" "llvm-ir" "dylib-llvm-aot")
+    set(_RULE_DRIVERS "vmla" "vulkan" "llvm" "dylib")
```

Comment:
This change should be mirrored at https://github.com/google/iree/blob/main/build_tools/bazel/iree_check_test.bzl#L20-L24, right?